src/sinol_make/commands/init/__init__.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import os
+import subprocess
+import tempfile
+
+from sinol_make import util
+from sinol_make.interfaces.BaseCommand import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Class for "init"
+    """
+
+    def get_name(self):
+        return "init"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Create package from the template',
+            description='Create package from predefined template. You have specify new task shortcut (like: '
+                        'sinol-make init abc )'
+        )
+        parser.add_argument('task_id', type=str, help='Id of the task to create')
+
+    @staticmethod
+    def download_template():
+        repo = 'https://github.com/sio2project/sinol-make.git'
+        package_dir = 'sinol-make/example_package'
+        tmp_dir = tempfile.mkdtemp()
+        subprocess.run(['git', 'clone', repo], cwd=tmp_dir, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        return os.path.join(tmp_dir, package_dir)
```

Comment:
I think there should be better error handling here, i.e. git's stderr should be directed to our stderr with subprocess.PIPE and we should check whether this command succeeds, otherwise print an error and exit cleanly.

I would also add `--depth 1` and maybe `-q` to the git command.

Reply:
The output of subprocesses should be automatically redirected to main process stdout/err.